scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -508,7 +508,7 @@ abstract class LocalScanner(
     /**
      * Return the scanner's raw result in a JSON representation.
      */
-    internal abstract fun getRawResult(resultsFile: File): JsonNode
+    abstract fun getRawResult(resultsFile: File): JsonNode
```

Comment:
Do you have to remove the `internal` here?

Reply:
Yes, unfortunately `internal` doe snot work with our custom `funTest` source set, only with the built-in `test` source set. Also see https://github.com/oss-review-toolkit/ort/blob/1fa1d749d65ee9a17dc3214d2abcb26aaf6978f8/build.gradle.kts#L174-L182